packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -188,6 +209,20 @@ class Chat extends Widget<Properties> {
     );
   }
 
+  _createTypingStartAction(): void {
+    this._typingStartAction = this._createActionByOption(
+      'onTypingStart',
+      { excludeValidators: ['disabled', 'readOnly'] },
+    );
+  }
+
+  _createTypingEndAction(): void {
+    this._typingEndAction = this._createActionByOption(
+      'onTypingEnd',
+      { excludeValidators: ['disabled', 'readOnly'] },
```

Comment:
I don't think we need this: there is no readOnly option in chat and there is no way to type in a disabled chat

Reply:
Thanks a lot. [788227a8e342ba4871b28d8c1988faa534237714]